content/en/docs/security/dos-mitigation.md

```diff
@@ -0,0 +1,408 @@
+---
+title : "DoS Mitigation"
```

Comment:
content/en/docs/dos-mitigation.md

DoS mitigation should be it's own section like how it is here: https://docs.libp2p.io/concepts/dos-mitigation/

Reply:
@p-shahi I collapsed the menu (I believe it is cleaner like this) which shows DoS mitigation without an additional click. Does this work?

Reply:
Sounds good!